src/test/java/core/basesyntax/service/impl/WriteToFileImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.WriteToFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriteToFileImplTest {
+    private static final String REPORT_FILE = "src/test/java/resources/report.txt";
+    private WriteToFile writer;
+    private List<String> report;
+
+    @Before
+    public void setUp() {
+        writer = new WriteToFileImpl();
+        report = new ArrayList<>();
+        report.add("banana,152" + System.lineSeparator());
+        report.add("apple,50" + System.lineSeparator());
+    }
+
+    @Test
+    public void writeToFile_filePathIsNull_notOk() {
+        try {
+            writer.writeToFile(null, report);
+        } catch (RuntimeException e) {
+            assertEquals("FilePath must be matched", e.getMessage());
+        }
+    }
```

Comment:
you can also use something like
```
RuntimeException exception = Assertions.assertThrows(RuntimeException.class, 
                                                     () -> writer.writeToFile(null, report));
assertEquals("FilePath must be matched", exception.getMessage());
```

Reply:
Can we use it in Junit 4?

Reply:
no, thanks
in Junit 4 we can use `@Test(expected = YourException.class)`